vm/tests/builtin.py

```diff
@@ -65,4 +65,25 @@
 else:
     assert False, "SyntaxError not raised"
 
+doc="isinstance"
+class A:
+    pass
+a = A()
+assert True, isinstance(1, (str, tuple, int))
+assert True, isinstance(a, (str, (tuple, (A, ))))
+
+try:
+    isinstance(1, (A, ), "foo")
+except TypeError:
+    pass
+else:
+    assert False, "TypeError not raised"
+
+try:
+    isinstance(1, [A, "foo"])
+except TypeError:
```

Comment:
https://github.com/go-python/gpython/blob/0c23b142f7b58bc925071df6ffdb04d7c112a21c/py/tests/libtest.py#L9

Please use `assertRaises` for this types of test. :)

Reply:
@corona10 Thanks! I created libtest.py to `vm/tests/` directory.